src/main/java/net/discordjug/javabot/listener/filter/MessageModificationStatus.java

```diff
@@ -0,0 +1,7 @@
+package net.discordjug.javabot.listener.filter;
+
+public enum MessageModificationStatus {
+    MODIFIED,
+    NOT_MODIFIED,
+    STOP_MODIFYING
```

Comment:
As filters would not _just_ be modifying, `STOP_PROCESSING` would be a better name.